FRBDK/Glue/Glue/FormHelpers/RightClickHelper.cs

```diff
@@ -2217,7 +2217,7 @@ public static void DeleteFolderClick(ITreeNode targetNode)
 
                     }
                 }
-                else if (targetNode.IsFolderInFilesContainerNode())
+                else if (targetNode.IsFolderInFilesContainerNode() || targetNode.IsChildOfGlobalContent())
```

Comment:
I believe this block of code should only act on folders, but `IsChildOfGlobalContent` may return `true` if the node is a files node. I'm investigating now...

Reply:
I checked with @zdman2022 and he explained that this method is `DeleteFolderClick` so this should only get called once the node in question has already been evaluated as a folder.